client/src/actions/index.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: `http://localhost:3300/api`,
```

Comment:
great job setting up your axios api options. A lot of people surprisingly enough don't know or use this amazing gift from the gods. Good job.